examples/test-LwIP-netconn/scpi_server.c

```diff
@@ -117,11 +113,10 @@ int SCPI_Error(scpi_t * context, int_fast16_t err) {
     iprintf("**ERROR: %ld, \"%s\"\r\n", (int32_t) err, SCPI_ErrorTranslate(err));
```

Comment:
In this sense, for me, it would be better to move iprintf to example error handlers.